src/pcapi/scripts/update_venue_creation_date.py

```diff
@@ -0,0 +1,30 @@
+import csv
+from typing import List
+
+from pcapi.repository import repository
+from pcapi.models import VenueSQLEntity, ApiErrors
+from pcapi.utils.logger import logger
+
+def update_venue_creation_date(file_path: str):
+    updated_venue_count = 0
+    venues_to_update = _read_venue_creation_date_from_file(file_path)
+    venue_ids_in_error = []
+
+    for venue_to_update in venues_to_update:
+        venue_id = int(venue_to_update[0])
+        venue = VenueSQLEntity.query.filter_by(id=venue_id).first()
```

Comment:
```suggestion
        venue = VenueSQLEntity.query.get(id=venue_id)
```

La ligne suivante va de toute façon crasher si l'identifiant est inconnu.

Reply:
je l'ai mis comme ça, `VenueSQLEntity.query.get(venue_id)`
j'avais une erreur sinon